test/verify/check-kdump

```diff
@@ -483,5 +506,145 @@ class TestKdumpNFS(KdumpHelpers):
+        self.assertActive(active=False)
+        customPath = "/var/crash2"
+
+        m.execute(f"mkdir -p {customPath}")
+        b.click("#kdump-change-target")
+        b.set_val("#kdump-settings-location", "local")
+        pathInput = "#kdump-settings-local-directory"
+        b.set_input_text(pathInput, customPath)
+        b.click(f"#kdump-settings-dialog button{self.primary_btn_class}")
+        b.wait_text("#kdump-change-target", f"locally in {customPath}")
+
+        # This requires a reboot
+        run_ansible_playbook(m, b, allow_failure=True)
+        m.reboot()
+
+        self.login_and_go("/kdump")
+        m.execute("until systemctl is-active kdump; do sleep 1; done")
+
+        # Verify that kdump runs and crashkernel is configured
+        b.wait_in_text("#app", "Service is running")
+        b.wait_in_text("#app", "192 MiB")
+        self.assertActive(active=True)
+
+        # Second run should be succeed as crashkernel is now set in the kernel cmdline
+        run_ansible_playbook(m, b)
+
+        conf = m.execute("cat /etc/kdump.conf")
+        self.assertIn("\n# Ansible managed\n", conf)
+        self.assertIn("\npath /var/crash2\n", conf)
+
+        self.crashKernel("stored in /var/crash2 as vmcore")
+        m.execute(f"until test -e {customPath}/127.0.0.1*/vmcore; do sleep 1; done", timeout=180)
+        self.assertIn("Kdump compressed dump", m.execute(f"file {customPath}/127.0.0.1*/vmcore"))
+
+        self.login_and_go("/kdump")
+        self.enableLocalSsh()
+
+        # Change back to SSH
+        b.click("#kdump-change-target")
+        b.set_val("#kdump-settings-location", "ssh")
+        sshInput = "#kdump-settings-ssh-server"
+        b.set_input_text(sshInput, "root@localhost")
+        sshKeyInput = "#kdump-settings-ssh-key"
+        pathInput = "#kdump-settings-ssh-directory"
+        b.set_input_text(sshKeyInput, "/root/.ssh/id_rsa")
+        b.set_input_text(pathInput, "/var/crash")
+        b.click(f"#kdump-settings-dialog button{self.primary_btn_class}")
+        # rebuilding initrd might take a while on busy CI machines
+        with b.wait_timeout(300):
+            b.wait_not_present(pathInput)
+        self.crashKernel("copied through SSH to root@localhost:/var/crash as vmcore", cancel=True)
+
+        b.wait_text("#kdump-change-target", "Remote over SSH")
+        run_ansible_playbook(m, b)
+        conf = m.execute("cat /etc/kdump.conf")
+        self.assertIn("\n# Ansible managed\n", conf)
+        self.assertIn("\nssh root@localhost\n", conf)
+        self.assertIn("\nsshkey /root/.ssh/id_rsa\n", conf)
+        self.assertIn("\npath /var/crash\n", conf)
+        self.crashKernel("copied through SSH to root@localhost:/var/crash as vmcore", cancel=True)
+
+
+@testlib.timeout(900)
+@testlib.skipOstree("kexec-tools not installed")
+@testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
+class TestKdumpAnsibleNFS(KdumpHelpers, packagelib.PackageCase):
```

Comment:
This can very well go into  TestKdumpNFS.testAnsible().

Reply:
Agreed, I hope the added onlyImage decorator can be added so it inherits the rest from the class